sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -240,11 +240,6 @@
   String CORE_AUTHENTICATOR_IGNORE_STARTUP_FAILURE = "sonar.authenticator.ignoreStartupFailure";
   String CORE_AUTHENTICATOR_CREATE_USERS = "sonar.authenticator.createUsers";
 
-  /**
-   * @since 3.6
-   */
-  String CORE_AUTHENTICATOR_UPDATE_USER_ATTRIBUTES = "sonar.security.updateUserAttributes";
```

Comment:
I suggest to keep the constant deprecated and to document the fact that feature is not supported anymore.